odh-notebook-controller/odh-notebook-controller/manager/manager.yaml

```diff
@@ -17,7 +17,7 @@ spec:
         runAsNonRoot: true
       containers:
         - name: manager
-          image: quay.io/opendatahub/odh-notebook-controller:latest
+          image: quay.io/opendatahub/odh-notebook-controller:1.6-9ae9413
```

Comment:
Couple of questions:

- Should we be updating https://github.com/opendatahub-io/odh-manifests/blob/master/odh-notebook-controller/gen_odh_manifests.sh#L8 and then generate the manifests?
-  Do we need to update this for every PR build in controller, or before ODH release ?

Reply:
Good questions.  

1. For this PR, I just wanted to replace the `latest` tag for odh-notebook-controller but I should update that tag in gen_odh-manifests.sh for consistency.  
2. I'll see if running gen_odh-manifests.sh makes any drastic changes in the manifest.  If it does then I'll keep the update push this update to the next release of ODH and we can follow-up on  https://github.com/opendatahub-io/kubeflow/issues/56 to streamline the process of syncing the manifests from the notebook-controller repo to odh-manifests